plenoptic/synthesize/eigendistortion.py

```diff
@@ -146,7 +146,7 @@ def synthesize(self,
                    max_steps: int = 1000,
                    p: int = 5,
                    q: int = 2,
-                   tol: float = 1e-8,
+                   tol: float = 1e-7,
```

Comment:
Added more reasonable tolerance, given that we're working with float32 by default